_source/_posts/2021-02-03-api-key-best-practices-and-examples.md

```diff
@@ -22,17 +22,18 @@ Today, we are going to create an API key for three different APIs. We will show,
 
 ## How to Obtain and Use an Okta API Token
 
-Okta provides APIs for authenticating users and validating tokens. To use Okta APIs you need to have an Okta account. If you don't have one already, [sign up for a free Okta account](https://developer.okta.com/signup/). 
+{% include setup/cli.md type="spa" loginRedirectUri="http://localhost:8080" %}
+
+Next, create an API token. Run `okta login` and open the resulting URL in your browser. Log in and go to **Security** > **API** > **Tokens**. Create a new token and store the value somewhere safe. Make sure you don't check it into GitHub! 
```

Comment:
@Dr-Phill-Edwards This prompted me to add a new `type="token"` to the CLI include. Now, you can replace these two paragraphs with:

```
{% include setup/cli.md type="token" %}
```

You can see what the rendered HTML looks like at http://developer.okta.com/blog/setup#api-token.